css/menu.css

```diff
@@ -114,39 +168,325 @@ header {
+    width: 60%;
+    display: flex;
+    justify-content: space-around;
+}
+
+footer nav .btm.btns a {
+    font-size: 1.6rem;
+}
+
+.copyright {
+    background-color: rgb(0, 4, 17);
+    color: rgb(225, 239, 230);
+    display: flex;
+    justify-content: flex-end;
+    width: 100%;
+    padding: 0 8%;
+}
+
+.copyright p {
+    font-size: 1.2rem;
 }
 
 /* mobile media query */
 @media (max-width: 428px) {
     /* header section */
-    .header-title {
+    header{
+        height:85vh;
+    }
+    header div {
+        height: 45vh;
+        display: flex;
+        justify-content: center;
+        flex-wrap: wrap;
+        align-items: flex-start;
+    }
+
+    header div h1 {
         width: 75%;
-        margin: 0 auto;
-        text-align: center;
-        font-size: 4rem;
+        padding-top:2%;
+    }
+
+    header div h1 a {
+        display: flex;
+        justify-content: center;
+        align-items: flex-start;
+        flex-wrap: wrap;
+        font-size: 3.2rem;
+        padding-bottom: 5%;
+        line-height: 1.25;
+    }
+
+    nav {
+        width: 70%;
+        display: flex;
+        justify-content: center;
+    }
+
+    .top.btns {
+        height: 20vh;
+    }
+
+    .top.btns a {
+        height: 5vh;
+        width: 100%;
+        display: flex;
+        justify-content: center;
+        font-size: 2.2rem;
+    }
+
+    nav div {
+        width: 50%;
+        height: 8vh;
     }
 
+    .social {
+        justify-content: space-evenly;
+        width: 80%;
+    }
+
+    .social a i {
+        font-size: 2rem;
+    }
+
+    /* Spacer */
+    .spacer{
+        height:17vh;
+    }
     /* menu */
     .menu {
         background-image: none;
+        background-color: rgb(225, 239, 230);
+        display: flex;
+        justify-content: center;
+        flex-wrap: wrap;
     }
 
-    /* footer section */
-    footer {
-        height: auto;
-        padding-top: 2.5vh;
-        font-size: 2rem;
+    .menu .menu-container{
+        background-color: rgb(225, 239, 230);
+        width:80%;
     }
+
+    .menu-container h2{
+        font-size: 4rem;
+        border-bottom: 1.6rem solid black;
+        padding-top: 20%;
+    }
+
+    .menu-container h2, .menu-section h3, .menu-item h4, .menu-item p{
+        color: rgb(0, 4, 17);
+        justify-content: center;
+        text-align: center;
+    }
+
+    .menu-section h3{
+        font-size: 4rem;
+        border: .5rem solid black;
+    }
+
+    .menu-item{
+        border-top: .5rem solid black;
+    }
+
+    .menu-item.one{
+        border: none;
+    }
+
+    .menu-item h4{
+        font-size: 3rem;
+    }
+
+    .menu-item p{
+        font-size:2.5rem;
+        display:flex;
+        justify-content: center;
+    }
+
+    .menu .menu-container .menu-section, .menu .menu-container .menu-section .menu-item, .menu .menu-container .menu-section .menu-item h4,.menu .menu-container .menu-section .menu-item p{
+        width:100%;
+        justify-content: center;
+    }
+
```

Comment:
Here if you had wrapped the flourless chocolate cake h4 and price in a div you could add the flex-wrap:wrap here to that div to make it look like the mock